### ISSUE TITLE
Function to set the default font

### ISSUE THREAD

**Issue by batblaster** [JujuAdams/scribble#61]
Hi,
would be nice if we can use a sort of draw_set_font, maybe, scribble_set_font function.
At the moment the only way to change the font is inside the text but when you pass from a language to another maybe is much more easy to use a function than add in all text the selected font.

Thanks

**Comment by DragoniteSpam** [comment]
You can use the `global.__scribble_default_font` variable to set the default font when creating a new scribble, or if you prefer, write a script that does it for you. Changes will only apply to newly created scribbles, though. Alternatively you can write a script to automatically append the [font] tags to the beginning and end of the string, which would cause the default font to appear to change on-the-fly.

**Comment by JujuAdams** [comment]
Thanks for your feature request!

Dragonite raises a couple good points. Writing a custom script that wraps around `scribble_draw()` and prefixes the desired font is what I'd expect most people would do; however, this does mean that `[]` won't neessarily reset the font as you'd expect.

It certainly is possible to set `global.__scribble_default_font`, though I wouldn't necessarily expect people to work that out for themselves. Doing this has the advantage that the dev can then use `[]` and it'll reset the font to the default.

I prefer this second solution so I propose making a function that handles the variable swapping for the user. When it comes to handling caching, it's just a case of adding the current default font to the cache string.

Funnily enough, I did originally have a function to set the default font, but I ended up stripping it out for the sake of simplicity!
